src/Raven.Server/Monitoring/Snmp/Objects/Database/5.2.X/2/DatabaseStorageDiskIosReadOperations.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using Lextm.SharpSnmpLib;
+using Raven.Server.Documents;
+
+namespace Raven.Server.Monitoring.Snmp.Objects.Database;
+
+public class DatabaseStorageDiskIosReadOperations : DatabaseScalarObjectBase<Gauge32>
+{
+    private static readonly Gauge32 Empty = new Gauge32(-1);
+
+    public DatabaseStorageDiskIosReadOperations(string databaseName, DatabasesLandlord landlord, int index)
+        : base(databaseName, landlord, SnmpOids.Databases.StorageDiskIoReadOperations, index)
+    {
+    }
+
+    protected override Gauge32 GetData(DocumentDatabase database)
+    {
+        if (database.Configuration.Core.RunInMemory)
+            return Empty;
```

Comment:
check if you should not return `null` instead

Reply:
It will return `NoSuchInstance`
```
{"Value":"NoSuchInstance"}
```